deps.edn

```diff
@@ -8,13 +8,13 @@
   clojure.java-time/clojure.java-time {:mvn/version "1.4.2"}
   commons-io/commons-io               {:mvn/version "2.16.1"}
   hiccup/hiccup                       {:mvn/version "1.0.5"}
-  org.apache.santuario/xmlsec         {:mvn/version "4.0.2"} ; use latest version and override transient dep from OpenSAML
-  org.cryptacular/cryptacular         {:mvn/version "1.2.7"} ; use latest version and override transient dep from OpenSAML
-  org.opensaml/opensaml-core          {:mvn/version "4.3.2"}
-  org.opensaml/opensaml-saml-api      {:mvn/version "4.3.2"}
-  org.opensaml/opensaml-saml-impl     {:mvn/version "4.3.2"}
-  org.opensaml/opensaml-xmlsec-api    {:mvn/version "4.3.2"}
-  org.opensaml/opensaml-xmlsec-impl   {:mvn/version "4.3.2"}
+  ;org.apache.santuario/xmlsec         {:mvn/version "4.0.2"} ; use latest version and override transient dep from OpenSAML
```

Comment:
mind deleting these commented out deps.

Reply:
I checked, and opensaml 5.1.3 still hasn't bumped them past this version, and even includes a version of xmlsec with a CVE. So I uncommented them to keep them.